### DISCUSSION TITLE
Set up the discussions

### DISCUSSION THREAD

**Discussion by 2Abendsegler** [2Abendsegler/GClh#1802] [General]
@capoaira @Ruko2010
Die Discussions lassen sich noch etwas einstellen. Beispielsweise kann man neue Kategorien machen und die vorgeschlagenen überarbeiten. Ihr könnt hier gerne auch etwas basteln, wenn ihr Lust habt. 😊 
Auch kann man das Announcement von mir noch mal überarbeiten.
In den nächsten Tagen werde ich auch die Discussions bei den Hilfen mit aufführen.

**Comment by 2Abendsegler** [comment]
... wir könnten auch noch ein paar Labels integrieren, die uns dabei helfen den Überblick zu wahren. 🤣

**Reply by capoaira** [reply]
Wir haben ja das Label "status: rejected"
Was hältst du von einem Label "proposal accepted"?
accepted hört sich auf Deutsch übersetzt irgendwie unhöflich an, aber ich kenne keine andere Übersetzung für "angenommen" 😅

**Comment by 2Abendsegler** [comment]
Link zur alten Discussion #1132

**Comment by 2Abendsegler** [comment]
Ein neues Label dafür finde ich gut. 

Wenn wir dem Teil einen Status verpassen wollen, dann sollten wir das Label im Einklang mit den anderen vielleicht auch mit "status:" beginnen lassen. Vielleicht können wir auch "approved" verwenden. Wenn wir das proposal noch weglassen, können wir das Label  noch allgemeiner auch für andere Dinge verwenden.

**Reply by capoaira** [reply]
Gute Idee, ich habe das Label erstellt (selbe Farbe wie rejected) und die Ideen gelabelt.

**Comment by 2Abendsegler** [comment]
Ich habe ein weiteres Label in grün angelegt: 
![grafik](https://user-images.githubusercontent.com/22332216/194555490-65e45157-60bd-4b33-9467-859d9be3643d.png)
Es kann bei den Fragen in den Discussions verwendet werden um beantwortete Discussions zu kennzeichnen. Auch wenn ja bei den Fragen ein answered gesetzt werden kann, macht es die Durchsicht der Fragen viel einfacher. Offen sind nur die, die kein grünes Label haben.
